### PR TITLE
Fixes #64

### LINKED ISSUE #64: As a citizen i should not be able to change my own settings
**Describe the bug**
Citizens can change their own settings

**To Reproduce**
Simply change the settings for a citizen, while logged in/authorized as a citizen.

**Expected behavior**
At der smides en "Authorization failed", når citizen forsøger.

**Actual behavior**
Ændringerne tillades og sker.

**Screenshots**

**Environment (please complete the following information):**
Running the dotnet project and database local, and generating rest calls through webbrowser (with swagger).

**Additional context**
Related to aau-giraf/api_client/issues/23

### PR DESCRIPTION
**Pull request by TobiasNielsen**

### CODE REVIEW

**Comment by giano574** [comment]
[Web API PR#69](https://trello.com/c/mTJ9dsSH/12-web-api-pr69)

**Comment by TobiasNielsen** [comment]
Obs.
Theres a lot of tests failing / skips which is an issue https://github.com/aau-giraf/web-api/issues/67
The tests that failed before our implementation, are those who fail after

File: GirafIntegrationTest/tests/userControllerTest.py [hunk 1]
```diff
@@ -233,3 +233,11 @@ def graatandGuardians(self, check):
         response = requests.get(Test.url + 'User/{0}/guardians'.format(self.graatandId),
                                 headers=auth(self.graatand)).json()
         ensureError(response, check)
+
+
+    @test(skip_if_failed=['loginAsKurt', 'GetKurtID'])
+    def citizenCantChangeSettings(self, check):
+        'Checking citizen can\'t change settings'
+        respons = requests.put(Test.url + 'User/{0}/settings'.format(self.kurtId), json=LargeData.allSettings,
```
Comment (giano574) [inline GirafIntegrationTest/tests/userControllerTest.py]:
```suggestion
        response = requests.put(Test.url + 'User/{0}/settings'.format(self.kurtId), json=LargeData.allSettings,
```

File: GirafIntegrationTest/tests/userControllerTest.py [hunk 2]
```diff
@@ -233,3 +233,11 @@ def graatandGuardians(self, check):
...
+                               headers=auth(self.kurt)).json()
+        ensureError(respons, check)
```
Comment (giano574) [inline GirafIntegrationTest/tests/userControllerTest.py]:
```suggestion
        ensureError(response, check)
```

**Comment by giano574** [comment]
Maybe add tests that show that guardians, departments, and super users CAN change citizen settings

File: GirafIntegrationTest/tests/userControllerTest.py [hunk 3]
```diff
@@ -233,3 +245,28 @@ def graatandGuardians(self, check):
...
+                               headers=auth(self.kurt)).json()
+        ensureError(response, check)
+
+    @test(skip_if_failed=['registerGunnar','loginAsLee', 'settingsMultiple'])
+    def superUserChangeCitizenSetting(self, check):
+        'Checking superUser can change citizens setting'
+        response = requests.put(Test.url + 'User/{0}/settings'.format(self.gunnarId), json=LargeData.updatedSettings1,
+                                headers=auth(self.lee)).json()
+        ensureSuccess(response, check)
+        check.equal(1, response['data']['theme'])
```
Comment (giano574) [inline GirafIntegrationTest/tests/userControllerTest.py]: Er det helt sikkert at den ikke kan have været én på forhånd?

Comment (TobiasNielsen) [reply GirafIntegrationTest/tests/userControllerTest.py]: Jep, testen kræver at testen "settingsMultiple" er kørt, denne test sætter theme til 3.
Dette ses af "skip_if_failed=[…  'settingsMultiple']"

**Pull request merged**